downloads/swan-lake-release-notes/swan-lake-2201.8.1/RELEASE_NOTE.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.8.1 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.8.1/
+active: 2201.8.1
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.8.1
+    - /downloads/swan-lake-release-notes/2201.8.1-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.8.1-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake Update 8 (2201.8.1)
+
+<em>Swan Lake 2201.8.1 is the first patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the language, standard library, and tooling.</em>
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to 2201.8.1 using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### New features
```

Comment:
```suggestion
### New features
```
Remove all empty sections.

Reply:
Done in: https://github.com/ballerina-platform/ballerina-dev-website/pull/8000/commits/9cb622b2b2823a237c2fd2759bea9c5f32c2671f